src/cache_cf.cc

```diff
@@ -625,22 +625,16 @@ Configuration::Parse()
 
     defaults_postscriptum();
 
+    if (unrecognizedDirectives)
```

Comment:
It may be tempting to throw even earlier, when unrecognizedDirectives constant is set above defaults_if_none() and defaults_postscriptum() calls. However, we should not do that. It is not obvious in the current code, but those two functions can also detect unrecognized directives and similar problems. Pending smooth reconfiguration changes will make this code look natural, removing the current "Why not throw earlier?" question.